tendrl/commons/objects/definition/master.yaml

```diff
@@ -21,6 +21,20 @@ namespace.tendrl:
     ceph-provisioner: "provisioner/ceph"
     ceph-installer: "provisioner/ceph"
   flows:
+    ConfigureMonitoring:
+      tags:
+        - "tendrl/node_$NodeContext.node_id"
+      atoms:
+        - tendrl.objects.Cluster.atoms.ConfigureMonitoring
+      help: "(Re)Configure Monitoring"
+      enabled: true
+      inputs:
+        mandatory:
+          - "Node[]"
```

Comment:
Why do you need input "Node[]"?

Reply:
That's the one that's used to target the flow to a particular node right?
Or the tags suffices?